### DISCUSSION TITLE
predict.py中修改model为lstm的问题

### DISCUSSION THREAD

**Discussion by wangzhen704** [KittenCN/stock_prediction#20] [General]
您好，用transformer进行train时花的时间太长就想试一下lstm，但一直报错
![image](https://github.com/KittenCN/stock_prediction/assets/22961987/4303b003-1f8a-46cd-9b64-635025996921)

**Comment by KittenCN** [comment]
lstm之前被我暂时放弃了，明天我看看，把它改出来

**Comment by KittenCN** [comment]
lstm现在正常可用了，我测试了下，看着train的速度快，但是要达到一定的进度，是需要大量的epoch的，总的耗时应该差不多，因为lstm的模型比较小，所以效率比transformer低
